pynest/nest/lib/hl_api_parallel_computing.py

```diff
@@ -86,3 +86,19 @@ def SetAcceptableLatency(port, latency):
 
     sps(latency)
     sr("/%s exch SetAcceptableLatency" % port)
+
+
+@check_stack
+def SetMaxBuffered(port, size):
+    """Set the maximum buffer size for a MUSIC port.
+
+    Parameters
+    ----------
+    port : object
```

Comment:
Yes, I suppose it should be `str`. But as you mentioned beneath, I was sticking to the prior convention.